cli.js

```diff
@@ -1,7 +1,4 @@
-import path from 'path';
-import generate from './index.js';
-import * as fs from './lib/fs.js';
-import * as stringify from './lib/stringify.js';
-import argv from './lib/cliArgs.js';
+import generate from './index';
```

Comment:
Unfortunately, removing the `.js` extension breaks the imports 😢 We are using ES modules rather than CommonJS modules, meaning the extension needs to be specified in the import path.

Reply:
I'd say add extensions back is the right way to go, that was there are no surprises